GettingStarted/ibmdbpy_GettingStarted_1-basics.ipynb

```diff
@@ -0,0 +1,2489 @@
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>sepal_length</th>\n",
+       "      <th>sepal_width</th>\n",
+       "      <th>petal_length</th>\n",
+       "      <th>petal_width</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>sepal_length</th>\n",
+       "      <td>0.685694</td>\n",
+       "      <td>-0.042434</td>\n",
+       "      <td>1.274315</td>\n",
+       "      <td>0.516271</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>sepal_width</th>\n",
+       "      <td>-0.042434</td>\n",
+       "      <td>0.189979</td>\n",
+       "      <td>-0.329656</td>\n",
+       "      <td>-0.121639</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>petal_length</th>\n",
+       "      <td>1.274315</td>\n",
+       "      <td>-0.329656</td>\n",
+       "      <td>3.116278</td>\n",
+       "      <td>1.295609</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>petal_width</th>\n",
+       "      <td>0.516271</td>\n",
+       "      <td>-0.121639</td>\n",
+       "      <td>1.295609</td>\n",
+       "      <td>0.581006</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "              sepal_length  sepal_width  petal_length  petal_width\n",
+       "sepal_length      0.685694    -0.042434      1.274315     0.516271\n",
+       "sepal_width      -0.042434     0.189979     -0.329656    -0.121639\n",
+       "petal_length      1.274315    -0.329656      3.116278     1.295609\n",
+       "petal_width       0.516271    -0.121639      1.295609     0.581006"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute the covariance matrix\n",
+    "idadf.cov()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Data selection and projection__\n",
+    "\n",
+    "When working with an IdaDataFrame, you can use classic slicing like with NumPy arrays and Pandas DataFrames."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Note: Use of an indexer*\n",
+    "\n",
+    "A warning would be raised by this line of code.\n",
+    "\n",
+    "> idadf_new1 = idadf[0:9] \n",
+    "\n",
+    " We want to select the first ten rows but \"Row order is not guaranteed if no indexer was given and the dataset was not sorted\".\n",
```

Comment:
https://pythonhosted.org/ibmdbpy/frame.html Original is:  IdaDataFrame.indexer

    The indexer attribute refers to the name of a column that should be used to index the table. This makes sense because dashDB is a column-based database, so row IDs do not make sense and are not deterministic. As a consequence, the only way to address a particular row is to refer to it by its index. If no indexer is provided, ibmdbpy still works but a correct row order is not guaranteed as far as the dataset is not sorted.

Reply:
If it's not a direct quote, just remove the quoting signs around it.

Reply:
Oh right you're talking about the actual quotation marks! I thought you were asking why it was so. 
The quotation is actually the error message which is prompted if you execute `idadf_new1 = idadf[0:9]` without having set an indexer. So I would rephrase it as: 
> We want to select the first ten rows but as the warning massage would say if you execute this without having set an indexer: "Row order is not guaranteed if no indexer was given and the dataset was not sorted".